src/utils/config.ts

```diff
@@ -133,6 +148,9 @@ export const loadConfig = async (program: Command) => {
         pwsh: config.prompt?.pwsh,
         nu: config.prompt?.nu,
       },
+      specs: {
+        path: config?.specs?.path,
```

Comment:
Let's also include the fig default location
```suggestion
        path: ["~/.fig/autocomplete/build", ...(config?.specs?.path ?? [])]
```

Reply:
<img width="790" alt="image" src="https://github.com/microsoft/inshellisense/assets/9245110/16842935-2b69-4f00-83f4-39cff3fad72e">


added it.